### DISCUSSION TITLE
Please add full Monero wallet to Ledger

### DISCUSSION THREAD

**Discussion by gushani** [LedgerHQ/ledger-live-desktop#4011] [Coin/Token request]
Please add Monero actual wallet to ledger upcoming update!
Thanks.

**Comment by funkyrelic** [comment]
I’d add to this: Monero over TOR capability and Haven Protocol support (based on Monero).

**Reply by TimmyBoi155** [reply]
YES

**Comment by TimmyBoi155** [comment]
And make sure to add support for running a local node and shielded addresses

**Comment by Intelli** [comment]
To be frank, this is long overdue.
